src/tasks/actions/thunks.ts

```diff
@@ -558,6 +558,25 @@ export const createTaskFromTemplate = (template: TaskTemplate) => async (
   }
 }
 
+export const retryTask = (taskID: string, runID: string) => async (
+  dispatch: Dispatch<Action | NotificationAction>
+) => {
+  try {
+    const resp = await api.postTasksRunsRetry({
+      taskID,
+      runID,
+    })
+
+    if (resp.status !== 200) {
+      throw new Error(resp.data.message)
+    }
+
+    dispatch(notify(copy.taskRetrySuccess()))
```

Comment:
let's add the task run id to the message. Something like `Task run ${runID} was successful`